editor/Storyboarding/EditorStoryboardLayer.cs

```diff
@@ -186,6 +186,8 @@ private void calculateSize()
             estimatedSize = 0;
 
             var exportSettings = new ExportSettings();
+            //removes overhead of optimising the osb in the export, potentially results in estimatedSize being slightly lower than the actual size
+            exportSettings.WriteToFile = false;
```

Comment:
Give it a name that's explicit about what it's doing rather than where it's (not) called from.